foundry/entity.d.ts

```diff
@@ -118,9 +118,11 @@ declare class Entity<D extends Entity.Data = Entity.Data> {
    * Render all of the Application instances which are connected to this Entity by calling their respective
    * {@link Application#render} methods.
    * @param force   - Force rendering
+   *                  (default: `false`)
    * @param context - Optional context
+   *                  (default: `{}`)
    */
-  render(force: boolean, context: any): void;
+  render(force?: boolean, context?: Application.RenderOptions): void;
```

Comment:
Is the context really the render options an application can get? Looks fine aside from that.

Reply:
The options are passed directly to Application.render

Reply:
Oh, wait, good catch